packages/exp/src/commands/build.js

```diff
@@ -39,10 +41,14 @@ export default (program: any) => {
     .option('--dist-p12-path <dist.p12>', 'Path to your Distribution Certificate P12.')
     .option('--push-p12-path <push.p12>', 'Path to your Push Notification Certificate P12.')
     .option('--provisioning-profile-path <.mobileprovision>', 'Path to your Provisioning Profile.')
+    .option('--public-url <url>', 'Url for an externally hosted app.')
```

Comment:
Maybe clarify this definition to make it clear this is a manifest URL? Something like “The URL of an externally hosted manifest (for self-hosted apps)”